global.d.ts

```diff
@@ -0,0 +1,3 @@
+// Use type safe message keys with `next-intl`
+type Messages = typeof import("./messages/en.json");
+declare interface IntlMessages extends Messages {}
```

Comment:
Is this the only configuration to support type-safe in `next-intl`?

Reply:
This is what the docs say!